### DISCUSSION TITLE
Gapfilling, lower interger

### DISCUSSION THREAD

**Discussion by brunorbqi** [opencobra/cobrapy#1399] [General]
I'm trying to do a Gapfilling but i get the following result : "RuntimeError: failed to validate gapfilled model, try lowering the integer_threshold" and when I tried to lower the integer_threshold as follows:

model.solver = 'gurobi'
gapfillSolution = cobra.flux_analysis.gapfill(model, universal, demand_reactions=True, exchange_reactions=True, integer_threshold=1e-09)

The outcome is: "TypeError: gapfill() got an unexpected keyword argument 'integer_threshold"

what I am doing wrong. Best Regards

**Comment by cdiener** [comment]
Sorry about that. This is a bug right now, because there is no way to set this parameter in the `gapfill` function. As a workaround, the following works instead of using the gapfill function:

```python
from cobra.flux_analysis.gapfilling import Gapfiller

gf = GapFiller(
    model=model,
    universal=universal,
    demand_reactions=True,
    exchange_reactions=True,
    integer_threshold=1e-9
)
filled = gf.fill(iterations=1)
```
